src/box/lua/config/applier/runtime_priv.lua

```diff
@@ -0,0 +1,64 @@
+local function apply_rt_access(config_module)
+    local configdata = config_module._configdata
+    local credentials = configdata:get('credentials') or {}
+    credentials.roles = credentials.roles or {}
+    credentials.users = credentials.users or {}
```

Comment:
It modifies the configuration table, so other callers of `config:get()` may receive the modified result. I think it is better to avoid it.

Reply:
Done.